components/push/src/communications.rs

```diff
@@ -404,17 +402,12 @@ impl Connection for ConnectHttp {
         if &self.options.sender_id == "test" {
             return Ok(false);
         }
-        let my_chans: Vec<String> = channels
-            .to_vec()
-            .into_iter()
-            .map(|s| Store::normalize_uuid(&s))
-            .collect();
-        log::debug!("Getting Channel List");
-        let remote = self.channel_list()?;
-        // verify both lists match. Either side could have lost it's mind.
+        let local_channels: HashSet<String> = HashSet::from_iter(channels.iter().cloned());
+        let remote_channels: HashSet<String> = HashSet::from_iter(self.channel_list()?);
```

Comment:
The equality of vectors was wrong here, since we were testing the content AND the order of the elements. Using a set is the proper way to do this.